Sources/MessagingInApp/MessagingInAppImplementation.swift

```diff
@@ -0,0 +1,98 @@
+import CioTracking
+import Common
+import Foundation
+import Gist
+
+internal class MessagingInAppImplementation: MessagingInAppInstance {
```

Comment:
All of this logic was copy/pasted from `MessagingInApp` class. To follow the same pattern that `CustomerIO` and `MessagingPush` classes follow with an `implementation` object.